docs/src/calibration_features.md

```diff
@@ -1,28 +1,59 @@
+- **Ensemble Kalman Inversion** uses an ensemble of model states to update the estimates of both the state and the parameters, and it is particularly robust against model errors.
+- **Unscented Kalman Inversion** is valuable to explore parameter uncertainties and their correlations.
+
+Additionally, for those who prefer straightforward optimization approaches, we integrate optimization tools from [`Optim.jl`](https://julianlsolvers.github.io/Optim.jl/stable/). 
 
 ## Configuration File
 
-To use the calibration tools, users must provide a configuration file detailing various elements of the calibration process:
+To use the calibration tools, it is crucial to provide a configuration file detailing all the elements needed for the calibration process. 
+The user must define:
+
+- **Priors**: the prior distributions of the parameters that need to be calibrated. The user must construct a dictionary in which each prior is specified by a `ParameterDistribution` object (see [parameter_distributions](https://clima.github.io/EnsembleKalmanProcesses.jl/dev/parameter_distributions/)). For each parameter, the definition of the prior requires the name of the parameter, the approximate mean and standard deviation of its distribution, and the interval in which it can take values.
+- **Observations**: The reference data against which simulations are compared. To do this, you can select the source of data ("perfect_model" or "file"), as well as the data names, data type, and the characteristics of the validation set for the calibrations.
+- **Calibration Process**: The methods and approaches used for calibration. In this part, the user must select the calibration method, as well as the variables that are needed for the process (e.g., number of iterations, batch size, random seed, etc.).
+- **Model Parameters setup**: Details about the parameters that affect the model's behavior, such as the model name, the characteristics of the grid, the aerosol initial distribution, the precipitation choice, etc.
+- **Statistics**: The statistical methods that can be applied on the data and simulation results to compare them.
+
+## How to run a calibration
+To be able to run a calibration, you need to import the `/KinematicDriver/CalibrateCMP` folder. 
+## Example
+To better understand how to build the configuration file and run calibrations in KiD, you can find a 1-dimensional calibration example in the `test/experiments/calibrations` folder.  In this example, we calibrate against the PySDM products of rain water mixing ratio (``r_r``) and cloud water mixing ratio (``r_l``), and KiD uses the [1-moment microphysics bulk scheme](https://clima.github.io/CloudMicrophysics.jl/dev/Microphysics1M/) to compute the same products. The parameters that are calibrated are ``\chi_v``, and ``\chi_a``, two parameters that appear in the terminal velocity (``v_{term}(r)``) and rain cross-section (``a(r)``) formulas of the bulk scheme:
```

Comment:
In this example, we calibrate the [1-moment microphysics bulk scheme](https://clima.github.io/CloudMicrophysics.jl/dev/Microphysics1M/) scheme in the perfect-model mode (by using data produced by the same method.)